package.json

```diff
@@ -63,15 +66,13 @@
     "@stiligita/keyframes": "^1.0.0-0",
     "@stiligita/react": "^1.0.0-0",
     "@stiligita/stylesheets": "^1.0.0-0",
-    "@types/react": "^16.0.31",
-    "@types/react-dom": "^16.0.3",
     "abcq": "^1.0.2",
     "ansi-to-rgb": "^1.0.0",
-    "load-asciicast": "^1.0.1",
-    "lodash": "^4.17.4",
-    "object-hash": "^1.2.0",
-    "react": "^16.0.0",
-    "react-dom": "^16.0.0",
+    "load-asciicast": "^2.0.0",
```

Comment:
Please publish a new version of load-asciinema:
https://github.com/marionebl/load-asciicast/pull/2/files